src/server/src/controllers/project.controller.ts

```diff
@@ -0,0 +1,41 @@
+import { Router } from 'express';
+import { ProjectService } from 'services/project.services';
+import { Request, Response } from 'express';
+import { createErrorResponse } from 'utils';
+
+const projectController = (router: Router): void => {
+  const projectService = new ProjectService();
+
+  router.get('/projects', async function (req: Request, res: Response) {
+    const projects = await projectService.getProjects();
+    console.log('RouterGet Projects:', projects);
+
+    // For each project, get the associated personnel, and then assign them to a casted
```

Comment:
this should actually be performed as part of a postgres JOIN operation.  It is extremely ineffecient to get all people from the table for each project which can be greatly mitigated by a JOIN.